src/dataset-builder/DatasetBuilder.ts

```diff
@@ -451,91 +450,83 @@ export const ValuesSelector = ({
       okButton: h(
         ButtonPrimary,
         {
-          disabled: errors,
-          tooltip: errors && Utils.summarizeErrors(errors),
-          onClick: withErrorReporting('Error creating dataset request')(async () => {
-            await DataRepo()
-              .snapshotAccessRequest()
-              .createSnapshotAccessRequest(
-                createSnapshotAccessRequest(
-                  name,
-                  researchPurposeStatement,
-                  snapshotId,
-                  cohorts,
-                  conceptSets,
-                  _.map(
-                    (valuesSet: HeaderAndValues<DatasetBuilderValue>) => ({
-                      domain: valuesSet.header,
-                      values: valuesSet.values,
-                    }),
-                    valueSets // convert from HeaderAndValues<DatasetBuilderType>[] to ValueSet[]
-                  )
-                )
-              );
-            onDismiss();
-          }),
+          href: '',
+          target: '_blank',
         },
-        ['Request access']
+        ['Continue to Form']
       ),
     },
     [
-      div([
-        div([
-          "A request of the dataset created will be generated and may take up to 72 hours for approval. Once approved you'll be notified by email. We'll send you a copy of this request",
-        ]),
-        h(FormLabel, { htmlFor: nameId, required }, ['Dataset name']),
-        h(ValidatedInput, {
-          inputProps: {
-            id: nameId,
-            'aria-label': 'Dataset name',
-            autoFocus: true,
-            placeholder: 'Enter a name',
-            value: name,
-            onChange: setName,
-          },
-        }),
-        h(FormLabel, { htmlFor: researchPurposeId, required }, ['Research purpose statement']),
-        h(ValidatedTextArea, {
-          inputProps: {
-            id: researchPurposeId,
-            'aria-label': 'Research purpose statement',
-            placeholder: 'Enter a research purpose statement',
-            style: {
-              marginTop: '1rem',
-              height: 200,
-            },
-            value: researchPurposeStatement,
-            onChange: setResearchPurposeStatement,
-          },
-        }),
+      div({ style: { lineHeight: 1.5 } }, [
+        'A request has been generated and may take up to 72 hours for approval. Check your email for a copy of this request. You’ll also be notified via email on approval of the request.',
       ]),
+      div(
+        {
+          style: {
+            backgroundColor: colors.accent(0.15),
+            color: colors.dark(),
+            border: `1px solid ${colors.accent(1.25)}`,
+            borderRadius: 10,
+            height: 150,
+            padding: '1rem',
+            marginTop: '1.5rem',
+            marginBottom: '2rem',
+          },
+        },
+        [
+          div({ key: 'important-container' }, [
+            h3({
+              key: 'important-heading',
```

Comment:
Its surprising to me that this should need a `key`. Usually, I would expect `key` to be necessary where we use loops or mapping to components. Have you been able to verify that this fixed the key error?

Reply:
Ya, I'm also surprised, but it ended up resolving the issues.